modin/experimental/engines/omnisci_on_ray/test/test_dataframe.py

```diff
@@ -317,20 +317,29 @@ def test_float32(self):
             True,
             False,
             ["col2"],
+            ["c2"],
+            [["col2", "col3"]],
+            {"col23": ["col2", "col3"]},
         ],
     )
+    @pytest.mark.parametrize("names", [None, [f"c{x}" for x in range(1, 7)]])
     def test_read_csv_datetime(
         self,
         engine,
         parse_dates,
+        names,
     ):
 
+        parse_dates_unsupported = isinstance(parse_dates, dict) or (
+            isinstance(parse_dates, list) and isinstance(parse_dates[0], list)
+        )
         eval_io(
             fn_name="read_csv",
-            md_extra_kwargs={"engine": engine},
+            md_extra_kwargs={"engine": None if parse_dates_unsupported else engine},
```

Comment:
this feels like it could be an extra test to make sure we raise some expected exception...

Reply:
In this cases Modin raises `ArrowEngineException` (raise of this exception is forced by setting `engine` to `arrrow` [here](https://github.com/modin-project/modin/blob/master/modin/experimental/engines/omnisci_on_ray/io.py#L302-L303)), but pandas raises `ValueError` that causes tests fails. But these tests looks redundant and were skipped in the last commit.